plugin/evm/vm.go

```diff
@@ -343,6 +343,8 @@ func (vm *VM) Initialize(
 			return fmt.Errorf("failed to unmarshal config %s: %w", string(configBytes), err)
 		}
 	}
+	vm.config.UpdateWithDefaults()
```

Comment:
I see why the call to UpdateWithDefaults is added. Fixing the config scheme for the VM seems a bit out of scope here. Can we set it to `defaultBlocksWindow` for all nodes instead?
This change doesn't impact nodes with pruning as they will already fail the API requests with "state not found" or similar.